docs/snippets/vue/login-form-with-play-function.mdx-3.mdx

```diff
@@ -22,21 +24,35 @@ export const Template = (args) => ({
     {Template.bind({})}
   </Story>
 
+<Canvas>
+  <Story name="Empty Form">
+    {Template.bind(())}
+  </Story>
+
   <Story
     name="Filled Form"
     play={async ({ canvasElement }) => {
       // Starts querying the component from its root element
-
       const canvas = within(canvasElement);
 
+      // 👇 Simulate interactions with the component
       await userEvent.type(canvas.getByTestId('email'), 'email@provider.com', {
         delay: 100,
       });
+
       await userEvent.type(canvas.getByTestId('password'), 'a-random-password', {
         delay: 100,
       });
-      // See https://storybook.js.org/docs/vue/essentials/actions#automatically-matching-args to learn how to setup logging in the Actions panel
+
+      // See https://storybook.js.org/docs/angular/essentials/actions#automatically-matching-args to learn how to setup logging in the Actions panel
```

Comment:
```suggestion
      // See https://storybook.js.org/docs/vue/essentials/actions#automatically-matching-args to learn how to setup logging in the Actions panel
```

Reply:
Fixed with the latest commit. Marking this as solved.